src/dlgPackageExporter.h

```diff
@@ -119,6 +120,7 @@ private slots:
     QString mPackageIconPath;
     QString mPackageConfig;
     QString mPlainDescription;
+    inline static bool mWritingZip;
```

Comment:
As you have `inline`d it - you can (since C++17) I *think*, initialise it here as well rather than having to at the start of the `.cpp` file...!
```suggestion
    inline static bool mWritingZip = false;
```
😁

Reply:
Good catch!